README.md

```diff
@@ -89,64 +89,80 @@ On Ubuntu do:
 ```bash
 sudo apt-get install libmysqlclient-dev
 ```
-3. Go to example/expq.chpl and inform database host, user, password. 
+3. Go to example/mysql_ex1.chpl and set database host, user, password. 
 4. Verify the mysql library path with ```bash mysql_config --cflags --libs ``` and edit Makefile.
 5. Go to repository folder and compile:
 ```bash
-make mysqlex
+make mysql_ex1
```

Comment:
A valid point, I think that including the required libraries within the code itself along with the `require` statements should be possible, so that those lengthy compiler options aren't needed; I'm currently looking into it.